tests/test_failure_propagator.py

```diff
@@ -1,15 +1,59 @@
 import pytest
 
-from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import (
+    acceptance,
+    tier1,
+    tier2,
+    tier3,
+    tier4a,
+    tier4b,
+    tier4c,
+)
 
 
 @tier1
-@pytest.mark.last
+@acceptance
+@tier2
+@tier3
+@tier4a
+@tier4b
+@tier4c
```

Comment:
what about pre and post upgrade markers?

Reply:
added